orchagent/bufferorch.cpp

```diff
@@ -520,17 +546,292 @@ task_process_status BufferOrch::processEgressBufferProfileList(Consumer &consume
     return task_process_status::task_success;
 }
 
+/*
+Input sample "BUFFER_PORT_CABLE_LENGTH_TABLE:"
```

Comment:
we need this in run time to pick proper profile on port speed change
if not APP_DB what storage can be used?